tests/components/netatmo/test_media_source.py

```diff
@@ -30,16 +31,55 @@ async def test_async_browse_media(hass):
             },
             1599152673: {
                 "id": "12346",
+                "type": "person",
                 "time": 1599152673,
                 "camera_id": "12:34:56:78:90:ab",
                 "snapshot": {
                     "url": "https://netatmocameraimage",
                 },
                 "message": "<b>Tobias</b> seen",
             },
+            1599152674: {
```

Comment:
For the future we could consider moving the test data to a json fixture file.

Reply:
Totally agree. It started small and felt ok in the test file but since it has grown I'll move it elsewhere.